fvtest/compilertest/tests/SingleBitContainerTest.cpp

```diff
@@ -0,0 +1,332 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2017, 2017
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+*******************************************************************************/
+
+
+#include "infra/BitVector.hpp"
+#include <string>
+#include "gtest/gtest.h"
+
+namespace {
+
+class SingleBitContainerTest : public :: testing :: Test {
+
+	protected:
+		TR_SingleBitContainer container;
+		TR_SingleBitContainer other;
+
+
+	virtual void SetUp() {
+		container.resetAll(1);
+		other.resetAll(1);
+	}
+
+
+};
+
+
+//****** restALL(int64_t m, int64_t) ******//
```

Comment:
Specify a variable for the second parm to match your code comments below:  `int64_t n`